core/translations/order.json

```diff
@@ -558,7 +558,7 @@
   "556": "reset__needed_to_form_a_group",
   "557": "reset__needed_to_recover_your_wallet",
   "558": "reset__never_make_digital_copy",
-  "559": "reset__num_of_share_holders_template",
+  "559": "reset__num_of_shares_how_many",
```

Comment:
Is this a backwards compatible change? If an older firmware uses newer translations it will expect a template but won't get one.

Reply:
Thank you, this is an omission actually, I decided to use both for now:
- `reset__num_of_share_holders_template` for TT and TS3
- `reset__num_of_shares_how_many` for TS5